client/signup/steps/difm-site-picker/index.tsx

```diff
@@ -76,19 +76,6 @@ export default function DIFMSitePickerStep( props: Props ): React.ReactElement {
 				}
 			)
 		);
-		//Skip domains step
```

Comment:
This part was redundant since the `domains` step is no longer a part of any DIFM flow.